xbmc/media/MediaType.h

```diff
@@ -37,6 +37,7 @@ using MediaType = std::string;
 #define MediaTypeTvShow           "tvshow"
 #define MediaTypeSeason           "season"
 #define MediaTypeEpisode          "episode"
+#define MediaTypeShoutcast        "shoutcast"
```

Comment:
So "internet radio" then? It is not "song", "album", "artist"... "tvshow"...
I will remove if it offends. I added it because there is another PR (not mine) that wants to have different Info lables for internet radio showing the station name and current song separately. This would have been a good step towards that.